src/debug_flags.cc

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-bool g_explaining = false;
```

Comment:
Per your commit message, one would argue that a library shouldn't have any global state at all, so multiple instances can be loaded and executed in the same application.